android/app/src/main/res/values/strings.xml

```diff
@@ -14,4 +14,16 @@
         <item>This is the medical car 3.</item>
         <item>This is the medical car 4.</item>
     </string-array>
+
+    <string-array name="serverOption">
+        <item>Offline mode</item>
+        <item>Online mode</item>
+    </string-array>
+
+    <string-array name="serverDescription">
+        <item>Use within the same department</item>
+        <item>Use from a different department</item>
+    </string-array>
+    
+    <string name="restrictBackButton">Restart the app to change server</string>
```

Comment:
After testing, it seems that the car also needs a restart to switch servers. I think it may also be good to notify the user that the car needs to restart in order to switch the server.

Reply:
Would this be notified to the user on the arduino side when back pressing on the main activity or what do you mean more exactly?

Reply:
Yes :+1:

Reply:
This is now done. The user will be notified of both app and car restart.